avaframe/com1DFAPy/runSimilaritySol.py

```diff
@@ -0,0 +1,335 @@
+"""
+    Run script for running python DFA kernel and similarity solution test
+"""
+import os
+import glob
+import time
+import copy
+import logging
+import numpy as np
+import math
+import scipy.stats as stats
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+# Local imports
+import avaframe.in3Utils.geoTrans as geoTrans
+import avaframe.in3Utils.initialiseDirs as inDirs
+import avaframe.in3Utils.initializeProject as initProj
+import avaframe.in3Utils.fileHandlerUtils as fU
+import avaframe.in2Trans.shpConversion as shpConv
+from avaframe.in1Data import getInput as gI
+import avaframe.com1DFAPy.com1DFA as com1DFA
+import avaframe.com1DFAPy.DFAtools as DFAtls
+import avaframe.com1DFAPy.DFAfunctionsCython as DFAfunC
+
+# from avaframe.DFAkernel.setParam import *
+import avaframe.out3Plot.plotUtils as pU
+import avaframe.out3Plot.makePalette as makePalette
+from avaframe.out1Peak import outPlotAllPeak as oP
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import avaframe.test3Num.simiSol as simiSol
+
+
+# +++++++++SETUP CONFIGURATION++++++++++++++++++++++++
+# log file name; leave empty to use default runLog.log
+logName = 'runSimilarityTest'
+
+# Load avalanche directory from general configuration file
+cfgMain = cfgUtils.getGeneralConfig()
+avalancheDir = cfgMain['MAIN']['avalancheDir']
+# set module name, reqiured as long we are in dev phase
+# - because need to create e.g. Output folder for com1DFAPy to distinguish from
+# current com1DFA
+modName = 'com1DFAPy'
+
+# Clean input directory(ies) of old work and output files
+initProj.cleanSingleAvaDir(avalancheDir, keep=logName)
+
+# Start logging
+log = logUtils.initiateLogger(avalancheDir, logName)
+log.info('MAIN SCRIPT')
+log.info('Current avalanche: %s', avalancheDir)
+
+# Load configuration
+simiSolCfg = os.path.join('test3Num', 'simiSol_com1DFACfg.ini')
+cfg = cfgUtils.getModuleConfig(com1DFA, simiSolCfg)
```

Comment:
Why are you reading it in com1DFAPy folder and not it the test3Num folder? 
I also noticed that you can put whatever module instead of com1DFA it still finds the ini file

Reply:
i am just using com1DFA to find the default file for this ini file